backend/inspirehep/migrator/tasks.py

```diff
@@ -320,8 +331,8 @@ def create_records_from_mirror_recids(self, recids):
     ignore_results=False,
     queue="migrator",
     acks_late=True,
-    retry_backoff=2,
-    retry_kwargs={"max_retries": 6},
+    retry_backoff=RETRY_BACKOFF,
+    retry_kwargs={"max_retries": MAX_RETRY_COUNT},
     autoretry_for=(OperationalError,),
```

Comment:
why not simply add it to the `autoretry`?

Reply:
As discussed IRL Micha misread code.